src/Sarif.Multitool.FunctionalTests/TestData/MessagesShouldEndWithPeriod/Invalid.sarif

```diff
@@ -253,12 +253,12 @@
             "runs[0].invocations[0].configurationNotifications[0].message.richText",
             "runs[0].invocations[0].toolNotifications[0].message.text",
             "runs[0].invocations[0].toolNotifications[0].message.richText",
-            "runs[0].resources.rules.TST0001.shortDescription.text",
-            "runs[0].resources.rules.TST0001.shortDescription.richText",
-            "runs[0].resources.rules.TST0001.fullDescription.text",
-            "runs[0].resources.rules.TST0001.fullDescription.richText",
-            "runs[0].resources.rules.TST0001.messageStrings.Default",
-            "runs[0].resources.rules.TST0001.richMessageStrings.Default"
+            "runs[0].resources.rules[0].shortDescription.text",
```

Comment:
>rules[0] [](start = 31, length = 8)

I assume this path has an array, even though the `rules` property above is a dictionary, because this file goes through the `PrereleaseCompatibilityTransformer`.

Reply:
that's right. i didn't care to tinker partially with this file. better to tackle all these comprehensively later when appropriate.

---
In reply to: [249156600](https://github.com/Microsoft/sarif-sdk/pull/1223#discussion_r249156600) [](ancestors = 249156600)